### PR TITLE
Titanic docs added 

### PR DESCRIPTION
**Pull request by arcAman07**
Titanic.md file added to the docs folder along with the updation of the make.jl file.
Hopefully it's ready for a release.
If any problems arise or needs to be done, I am ready to fix it asap.

### CODE REVIEW

**Comment by CarloLucibello** [comment]
should be added also to the README

**Comment by arcAman07** [comment]
> should be added also to the README

Yeah I will just fix that open the PR with the necessary changes.

File: src/Titanic/Titanic.jl [hunk 1]
```diff
@@ -116,13 +115,13 @@ julia> using MLDatasets: Titanic
 julia> target = Titanic.targets();
 
 julia> summary(target)
-"1×891 Matrix{Float64}"
-     
+"1×891 Matrix{}"
+```
 """
```
Comment (CarloLucibello) [inline src/Titanic/Titanic.jl:121]: the should be no blank line between the function definition and its docstring

Comment (arcAman07) [reply src/Titanic/Titanic.jl:121]: Oohk that's what I was missing the whole time. Will fix it in a couple of hours .

**Comment by arcAman07** [comment]
I guess it's ready to be merged. All issues fixed

**Review by johnnychen94 — APPROVED** [review]

**Pull request merged**